src/context/mod.rs

```diff
@@ -613,12 +613,23 @@ impl Context {
     pub fn open_key(&self, key: &RedisString) -> RedisKey {
         RedisKey::open(self.ctx, key)
     }
+    
+    #[must_use]
+    pub fn open_key_with_flags(&self, key: &RedisString, flags: &Vec<KeyFlag>) -> RedisKey {
```

Comment:
Why not using `bitflags!` for the flags attribute. Currently it is easy to make mistakes such as passing the same flag twice.

Reply:
Also it is more performant